Theme/Theme/Theme.swift

```diff
@@ -91,6 +91,22 @@ public struct Theme {
         }
     }
     
+    public struct UIColors {
+        public private(set) static var textPrimary = ThemeAssets.textPrimary.color
+        public private(set) static var accentColor = ThemeAssets.accentColor.color
+        public private(set) static var background = ThemeAssets.background.color
+
+        public static func update(
+            textPrimary: UIColor = ThemeAssets.textPrimary.color,
+            accentColor: UIColor = ThemeAssets.accentColor.color,
+            background: UIColor = ThemeAssets.background.color
+        ) {
+            self.textPrimary = textPrimary
+            self.accentColor = accentColor
+            self.background = background
+        }
+    }
+
```

Comment:
@rnr Why it's needed to create a new struct UIColors when we already have `uiColor()` extension written on `Colors`?

Even if we want to define categorical UIColors, I guess it would be nice to have all the values that Colors have and also it would be nice if you can remove `uiColor()` extension so we will have a single source. Thoughts?